mathesar_ui/src/sections/table-view/constraints/TableConstraint.svelte

```diff
@@ -31,7 +37,12 @@
     }
   }
 
-  $: columnSummary = constraint.columns.join(', ');
+  $: ({ columnsDataStore } = $tabularData);
+  $: columnsData = $columnsDataStore;
+  $: columnsInTable = columnsData.columns;
+  $: columns = columnsInTable.filter((c) => constraint.columns.includes(c.id));
+  $: columnNames = columns.map((columnInConstraint) => columnInConstraint.name);
+  $: columnSummary = columnNames.join(', ');
```

Comment:
@seancolsen We might want to implement some common methods as part of each store.

In this case `getByIds` in `columnsDataStore`, which accepts ids and returns related columns.

Reply:
@pavish Cool. This should now be addressed in ac473ca8e64f9b22c918a919309de24ddda8ffde

Reply:
Looks good